contracts/GoodGhosting_Polygon.sol

```diff
@@ -0,0 +1,144 @@
+// SPDX-License-Identifier: UNLICENSED
+
+pragma solidity 0.6.11;
+
+import "@openzeppelin/contracts/access/Ownable.sol";
+import "@openzeppelin/contracts/utils/Pausable.sol";
+import "@openzeppelin/contracts/math/SafeMath.sol";
+import "@openzeppelin/contracts/token/ERC20/IERC20.sol";
+import "./aave/ILendingPoolAddressesProvider.sol";
+import "./aave/ILendingPool.sol";
+import "./aave/AToken.sol";
+import "./aave/IncentiveController.sol";
+import "./quickswap/IRouter.sol";
+import "./GoodGhostingWhitelisted.sol";
+import "./GoodGhosting.sol";
+
+/**
+ * Play the save game.
+ *
+ */
+
+contract GoodGhosting_Polygon is Ownable, Pausable, GoodGhosting {
+    IncentiveController public incentiveController;
+    IRouter public router;
+    IERC20 public immutable matic;
+    IERC20 public immutable quick;
+
+    /**
+        Creates a new instance of GoodGhosting game
+        @param _inboundCurrency Smart contract address of inbound currency used for the game.
+        @param _lendingPoolAddressProvider Smart contract address of the lending pool adddress provider.
+        @param _segmentCount Number of segments in the game.
+        @param _segmentLength Lenght of each segment, in seconds (i.e., 180 (sec) => 3 minutes).
+        @param _segmentPayment Amount of tokens each player needs to contribute per segment (i.e. 10*10**18 equals to 10 DAI - note that DAI uses 18 decimal places).
+        @param _earlyWithdrawalFee Fee paid by users on early withdrawals (before the game completes). Used as an integer percentage (i.e., 10 represents 10%).
+        customFee
+        @param _dataProvider id for getting the data provider contract address 0x1 to be passed.
+     */
+    constructor(
+        IERC20 _inboundCurrency,
+        ILendingPoolAddressesProvider _lendingPoolAddressProvider,
+        uint256 _segmentCount,
+        uint256 _segmentLength,
+        uint256 _segmentPayment,
+        uint256 _earlyWithdrawalFee,
+        uint256 _customFee,
+        address _dataProvider,
+        address _incentiveController,
+        IRouter _router,
+        IERC20 _matic,
+        IERC20 _quick,
+        bytes32 merkleRoot_
+    )
```

Comment:
I'd suggest you pass the new parameters at the end, to make it more similar to the GoodGhosting contract (for similarity and maintainability reasons, only).

```suggestion
    constructor(
        IERC20 _inboundCurrency,
        ILendingPoolAddressesProvider _lendingPoolAddressProvider,
        uint256 _segmentCount,
        uint256 _segmentLength,
        uint256 _segmentPayment,
        uint256 _earlyWithdrawalFee,
        uint256 _customFee,
        address _dataProvider,
        bytes32 merkleRoot_,
        address _incentiveController,
        IRouter _router,
        IERC20 _matic,
        IERC20 _quick
    )
```